### DISCUSSION TITLE
Problems with simulation start times

### DISCUSSION THREAD

**Discussion by liu-13** [NCAR/hrldas#230] [Q&A]
I used ERA5-land data for driving the Noah-MP land surface model. The start time is set to 12:00 on July 28, 2016 and no errors are reported when running Prepare_ERA5-Land_Forcing.sh and extract_era5_init.py. But after I run create_forcing.exe, the first momentary LDASIN file generated, the 2016072812.LDASIN_DOMAIN1 file, does not have the VEGFRA and LAI variables in it. And the Q2 obtained from later simulation moments has a large number of default values. Whereas, if the simulation starts at 00:00, everything works fine and does not have these problems with the simulation starting at 12:00. I don't know what's the problem, please help, thanks!The following two graphs show the Q2MV at the same moment from the simulation starting at 12:00 and 00:00, respectively.
![bc3bf9ede5b73b7ebdd433f1185c5b20](https://github.com/user-attachments/assets/24004c8b-b6bb-4a3c-af52-9c2ef35c77c8)
![ad7de1d5150e060945e82c7555c76a9d](https://github.com/user-attachments/assets/440eeacd-82af-4c05-b313-5c830f8c158b)

**Comment by cenlinhe** [comment]
which dynamic vegetation option did you use? If you use the option that requires time series of vegetation fraction and LAI from input files, you will need to add the vegetation fraction and LAI variables to every forcing files based on the time. The current ERA5 preprocessing scripts and create_forcing.exe will not do this. They only generate the required meteorological forcing variables. 
Also, please double check your generated forcing files and HRLDAS_setup files to make sure the variables inside the files have reasonable values.

**Reply by liu-13** [reply]
It has been checked that the forcing data and the HRLDAS_setup file data are reasonable. However, I found a problem that Q2MV and Q2MB in the LDASOUT file at the initial moment of the simulation time are default values, 2m temperature is not. And, the Q2 in the later moments is also not the default value. What is the reason for this? How can I get the Q2 of the first moment not to be the default value?

**Reply by tslin2** [reply]
it could be because the day (12:00) and night (00:00) have different atmospheric conditions that lead to different latent heat fluxes, affecting Q2

**Reply by tslin2** [reply]
Note that the model requires climate forcing input at UTC time, not the local time

**Reply by cenlinhe** [reply]
Also, the first time point of the output has the value from HRLDAS_setup file and Noah-MP initialization module to initialize those variables, which has not gone through any Noah-MP physics calculation before outputing the first time point value. I am not sure what you meant by "default value". Q2 will not be the same as the specific humidity read from your forcing file, because your forcing variables will be at the ZLVL forcing level (by default it is 10-m) and Noah-MP will re-calculate the values at 2-m height level.

**Reply by liu-13** [reply]
> it could be because the day (12:00) and night (00:00) have different atmospheric conditions that lead to different latent heat fluxes, affecting Q2

I use UTC time.It is the ERA5-land forcing data that is utilized

**Reply by liu-13** [reply]
> default value

”default value“ means -9999e+03.

**Reply by tslin2** [reply]
Do you use same HRLDAS_setup file for two runs? Or different ones?

**Reply by liu-13** [reply]
> Do you use same HRLDAS_setup file for two runs? Or different ones?

different ones. However, I found that as long as the initial moment is simulated, regardless of whether the initial moment is 12:00 or 00:00, the Q2MV and Q2MB in the LDASOUT file for the first simulated moment I get are -9999e+03. but they are all normal in the LDASIN file.

**Reply by cenlinhe** [reply]
Ok, to answer your previous questions:
1. because you used differnet HRLDAS_setup file for your two simulations starting at different times, your results will be different.
2. The Q2MV and Q2MB are diagnostic variables from model which are set to the default missing value -9999 at the very initial/starting time point in the output file, because the model has not executed any physics to compute Q2MV and Q2MB yet.
3. The Q2 from model output will be different from the specific humidity in the forcing (even if you use Q2 for the forcing variable), because the  specific humidity  in the forcing is the value at the model forcing level (ZLVL set in the namelist.hrldas). The model will further use M-O theory to compute a new Q2 at the 2-m level (as I explained earlier).

**Reply by liu-13** [reply]
> Ok, to answer your previous questions:
> 
> 1. because you used differnet HRLDAS_setup file for your two simulations starting at different times, your results will be different.
> 2. The Q2MV and Q2MB are diagnostic variables from model which are set to the default missing value -9999 at the very initial/starting time point in the output file, because the model has not executed any physics to compute Q2MV and Q2MB yet.
> 3. The Q2 from model output will be different from the specific humidity in the forcing (even if you use Q2 for the forcing variable), because the  specific humidity  in the forcing is the value at the model forcing level (ZLVL set in the namelist.hrldas). The model will further use M-O theory to compute a new Q2 at the 2-m level (as I explained earlier).

Thank you. But I also found a problem, with normal forcing data, no matter if the simulation starts from 00:00 or 12:00, the Q2MB and Q2MV in the LDASOUT file for the first moment of the simulation is -9999, and in the whole simulation area it's -9999. but not for the subsequent moments, it's just normal. I don't know what is the reason for this.

**Reply by cenlinhe** [reply]
the reason is my 2nd point mentioned above.

**Reply by liu-13** [reply]
> 2\. The Q2MV and Q2MB are diagnostic variables from model which are set to the default missing value -9999 at the very initial/starting time point in the output file, because the model has not executed any physics to compute Q2MV and Q2MB yet.

I'm sorry, I read it wrong. Thank you for your answer, I understand.
